### PR TITLE
Release notes for 1.6.0-alpha.1

### PR DESCRIPTION
**Pull request by justinsb**
Proposed release notes for 1.6.0-alpha.1

cc @geojaz @yissachar @chrislovecnm @kris-nova

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/2237)

### CODE REVIEW

File: docs/releases/1.6.0-alpha.1.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+Note that the annotation form is ignored.  To schedule a pod on the master, the toleration must be updated
+and moved from an annotation to the field.
+
+* A new label for nodes, mirroring the toleration, is added and is now preferred: `node-role.kubernetes.io/master=`
+(`node-role.kubernetes.io/master` with an empty value).  `kubernetes.io/role=master` is still present, but
+the `node-role.kubernetes.io/<role>=` form is preferred.  `kubernetes.io/role=node` and `node-role.kubernetes.io/node=`
+are also present.
+
+* Rolling update to 1.6 does not succeed because new kube-dns pods mount a configmap with an optional volume map,
+but that is enforced by the kubelets, which are upgraded after the master.
```
Comment (yissachar) [inline docs/releases/1.6.0-alpha.1.md]: Move to a `Known Issues` section to make it stand out more for people just scanning?

File: docs/releases/1.6.0-alpha.1.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+## 1.6.0-alpha.1
+
+1.6.0-alpha.1 is a prerelease early-access of kops 1.6, which is the release with full support for kubernetes 1.6.
+This version of kops & kubernetes has not yet undergone extensive validation, and there will be improvements
+made before release of kops 1.6.0.
+
+This is not a full set of release notes, but rather a summary of the highest impact changes in the 1.6 release:
+
+* kops is not yet recommending etcd3, pending issues such as HA upgrade support.
```
Comment (yissachar) [inline docs/releases/1.6.0-alpha.1.md]: This point could use some re-work to explain that etcd3 is new in kubernetes 1.6, but that kops is still defaulting to etcd2 and etcd3 is run-at-your-own-risk.

**Comment by kris-nova** [comment]
@yissachar I am going to merge this so we can have some talking points today at the conference, but I will open up a PR with your concerns! I hope that is okay.

**Comment by kris-nova** [comment]
PR here: https://github.com/kubernetes/kops/pull/2239

**Pull request merged**